lib/tests/streamlit/elements/chat_test.py

```diff
@@ -235,8 +233,8 @@ def test_session_state_rules(self):
             st.session_state.my_key = "Foo"
             st.chat_input("Placeholder", key="my_key")
 
-        self.assertEqual(
-            SESSION_STATE_WRITES_NOT_ALLOWED_ERROR_TEXT,
+        self.assertIn(
+            "Values of the widget with key",
```

Comment:
instead of duplicating the string across the files, I suggest to add a new function 

```python
def get_session_states_write_not_allowed_error_msg(key: str) -> str:
  return f'Values of the widget with key "{key}" cannot be set using `st.session_state`.'
```
and use that in the `utils.py` and the test files

Reply:
I'm not a big fan of building abstractions in the main library purely for usage in testing... I think it's totally fine to update the string in the test for the very uncommon situation in which we change an error text.

Reply:
ok, fair point